File: uri 1010.c
#include <stdio.h>
 
int main() {
    int a,b,m,n;
    float x,y,result;
    scanf("%d %d %f",&a,&b,&x);
    scanf("%d %d %f",&m,&n,&y);
    result = b*x + n*y;
    printf("VALOR A PAGAR: R$ %.2f\n",result);
 
    return 0;
}

File: uri 1010.py
line1 = input().split(" ")
line2 = input().split(" ")
product1, unit1, price1 = line1
product2, unit2, price2 = line2
pay = (int(unit1) * float(price1)) + (int(unit2) * float(price2))

print('VALOR A PAGAR: R$ %0.2f'%pay)
